File: DSA/Vikash_kumar/Day_1_Question_2.py
#Day1 Question2:-
arr = []
arr = input("Enter the numbers with the space : ").split(" ")
k = int(input("Enter the value of k : "))
count = 0

for i in range(len(arr)):
    for j in range(i+1, len(arr)):
        if (int(arr[i]) + int(arr[j])) % k == 0:
            count += 1
print(f"The number of pairs where 1st<2nd and divisible by {k} = {count}")

File: DSA/SparshJain/Day_1_Question_2.cpp
#include<iostream>
using namespace std;

int Pairs(int arr[],int size,int k){
    int count=0;
    for(int i=0;i<size;i++){
        for(int j=0;j<size;j++){
            if((arr[i]+arr[j])%k==0 && i<j && i!=j){
                count++;
            }
        }
    }
    return count;
}

int main(){
    int arr[6]={1,3,2,6,1,2};
    int k;
    cout<<"Enter the value of k:";
    cin>>k;
    int count= Pairs(arr,6,k);
    cout<<"Number of (i,j) pairs divisible by k are:"<<count;
}